src/core/linker.cpp

```diff
@@ -292,7 +306,9 @@ bool Linker::Resolve(const std::string& name, Loader::SymbolType sym_type, Modul
     return false;
 }
 
-void* Linker::TlsGetAddr(u64 module_index, u64 offset) {
+void* Linker::TlsGetAddr(u64 module_index, u64 offset) 
+{
+#ifdef __x86_64__
```

Comment:
You should probably just stub `GetTcbBase()` for now if anything, not the whole thing here.

Reply:
Done